### PR TITLE
Add python-nectar.

### PR DESCRIPTION
**Pull request by bowlofeggs**

### CODE REVIEW

File: rpms/python-nectar/python-nectar.spec [hunk 1]
```diff
@@ -0,0 +1,214 @@
+Name:           python-nectar
+Version:        1.5.1
+Release:        2%{?dist}
+BuildArch:      noarch
+
+License:        GPLv2
```
Comment (jeremycline) [inline rpms/python-nectar/python-nectar.spec]: GPLv2+

File: rpms/python-nectar/python-nectar.spec [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+Group:          Development/Tools
+Summary:        A download library that separates workflow from implementation details
```
Comment (jeremycline) [inline rpms/python-nectar/python-nectar.spec:8]: Heh

Comment (bowlofeggs) [inline rpms/python-nectar/python-nectar.spec:8]: Hehe

File: rpms/python-nectar/python-nectar.spec (hunk 1 above)
Comment (bowlofeggs) [inline rpms/python-nectar/python-nectar.spec]: Fixed.

**Pull request merged**